content/blog/2025/oss-update.md

```diff
@@ -0,0 +1,63 @@
+---
+title: New Features for OddContrast
+sub:
```

Comment:
Missing a subtitle.

I think the file name could also be specific to the contents, since we're likely to do more oss updates in 2025.